10.9-libcxx/stable/main/finkinfo/devel/meson-py.info

```diff
@@ -0,0 +1,103 @@
+# -*- coding: ascii; tab-width: 4 -*-
+Info2: <<
+Package: meson-py%type_pkg[python]
+Version: 0.62.2
+Revision: 1
+Type: python (3.7 3.8 3.9 3.10)
+Depends: python%type_pkg[python], ninja (>= 1.7-1)
+BuildDepends: fink (>= 0.24.12), setuptools-tng-py%type_pkg[python]
+
+Source: https://files.pythonhosted.org/packages/source/m/meson/meson-%v.tar.gz
+Source-Checksum: SHA256(a7669e4c4110b06b743d57cc5d6432591a6677ef2402139fe4f3d42ac13380b0)
+
+PatchScript: <<
+    #!/bin/sh -ev
+    HDF5INC="$(ls -d %p/opt/hdf5.v1.12/include 2> /dev/null || echo '')"
+    perl -pi -e 's|(/bin/env python)(3)|${1}%type_raw[python]|;' *.py mesonbuild/*.py tools/*.py test*/*/*/*.py
+    perl -pi -e "s|(python)(3)(')|\${1}%type_raw[python]\${3}|;" test\ cases/python/[1345]*/meson.build
+    perl -pi -e "s|(python)(3)(')|\${1}-%type_raw[python]\${3}|;" test\ cases/python/2*/meson.build
+    if [ -n "$HDF5INC" ]; then
+        HDF5LIB=${HDF5INC%%/include}/lib
+        perl -pi -e "s|(h5c =)|add_project_arguments('-I${HDF5INC}', language: ['c', 'cpp'])\n\$1|;" "test cases/frameworks/25 hdf5/meson.build"
+        perl -pi -e "s|(h5c =)|add_project_link_arguments('-L${HDF5LIB}', language: ['c', 'cpp'])\n\$1|;" "test cases/frameworks/25 hdf5/meson.build"
+    fi
+<<
+
+CompileScript: <<
+    %p/bin/python%type_raw[python] setup.py build
+<<
+
+InstallScript: <<
+    %p/bin/python%type_raw[python] setup.py install --root %d
+    mv %i/bin/meson %i/bin/meson-py%type_pkg[python] 
+    mv %i/share/man/man1/meson.1 %i/share/man/man1/meson-py%type_pkg[python].1 
+    perl -pi -e 's|(/usr)(/bin/python)(3)|%p${2}%type_raw[python]|;' %i/share/polkit-1/actions/com.mesonbuild.install.policy
+    perl -pi -e 's|(/usr)(/bin/meson)|%p${2}-py%type_pkg[python]|;' %i/share/polkit-1/actions/com.mesonbuild.install.policy
+    mv %i/share/polkit-1/actions/com.mesonbuild.install.policy %i/share/polkit-1/actions/com.mesonbuild.install.policy-py%type_pkg[python]
```

Comment:
Do you actually need to install a polkit file at all? AFAICT fink doesn't have polkit available anyway.

Reply:
Good point, this was left from the initial version; the only other polkit file present there on my system was installed by `dpkg`.